src/olympia/reviewers/forms.py

```diff
@@ -497,8 +514,54 @@ def clean(self):
                 raise ValidationError(
                     'Multiple policies selected with different cinder actions.'
                 )
+
+        if self.helper.actions.get(selected_action, {}).get('delayable'):
+            if 'delayed_rejection' not in self.data:
```

Comment:
It's not totally clear to me how this wouldn't be handled by setting the fields to required (which you do) or using required on the html?

Reply:
mmm looks like we already manipulate `required` in `is_valid()` above for various fields depending on the action so yeah, I'll try doing that there and drop the custom `ValidationError`

Reply:
Actually, can't do that because `NullBooleanField` accepts missing values as `None` without raising, even if the field is `required`. I added a comment and stopped hardcoding the error message in 407703a7f58465cf9f570e68eacfdc2534b664d6